charts/cloudcasa/v0.1.0/questions.yml

```diff
@@ -0,0 +1,14 @@
+labels:
+  io.rancher.certified: partner
+  rancher_min_version: 2.3.0
+  rancher_max_version: 2.x
```

Comment:
Please remove these, they are not necessary. This repository is for Helm 3 charts only, and we only support Helm 3 in Rancher v2.4+ so your chart will be made available in all Rancher releases that support this catalog.

Reply:
Thanks @PennyScissors. We'll make that change.